### DISCUSSION TITLE
Crazyflie Bolt - Bypass Onboard DC/DC Converter

### DISCUSSION THREAD

**Discussion by taliaxu09** [bitcraze/discussions#906] [Q&A]
Hello, we are trying to use a 18650 battery to power the crazyflie bolt, but it seemed like the board is not working properly until we increase the supply battery to about 7.2 V (when we debug with a power supply).

After looking at the document, it seems like we have to carry out this step "Onboard 5V DC/DC @500mA for voltages above 5V, else passthrough.", but looking at the schematics, there doesn't seem to be an easy way to do this that doesn't involve cutting some traces, but cutting the traces seem to be pretty difficult on this board since it's quite compact. Directly connecting VCOM (or VCOM_EXP) to a 3.2V supply doesn't seem to help. I was wondering if this is the right track (bypassing the DC/DC converter) and if there is an easier way to carry out this task?

Also would like to verify that the DC/DC converter to bypass is indeed the TPS62170 (U4) and nothing else.

Thank you for your help!

**Comment by tobbeanton** [comment]
I guess you hit the current limit and the DC/DC shuts down?

The best solution we found so far is to feed the USB input with the battery voltage as well. This will prevent the DC/DC from shutting down when current goes high or voltage is low. It requires a small patch as shown in the picture, connecting VBAT to VUSB. There are some drawbacks though:

1. Don't connect anything more than 1-cell as this might destroy the regulators.
2. Don't have USB and battery connected at the same time.

![image](https://github.com/bitcraze/discussions/assets/6593186/5d845fdd-9091-48a4-af07-990144674472)

**Reply by taliaxu09** [reply]
Thank you for your response, could you elaborate on what "I guess you hit the current limit and the DC/DC shuts down" means? Using a power supply the board never consumes more than 100 mA at any time, and I think the DC/DC can handle up to 600 mA.

If we do not do any change to the board, could we have powered it with 18650 voltage supplying around 3.7 V? If so maybe we were doing something wrong?

**Reply by tobbeanton** [reply]
The current limit can be hit if you e.g. use 1-cell with a LED-ring attached at full power and if that is not the case it should work with 1-cell battery except these findings:

- When the Bolt 1.1 is running on one cell battery (3-4.2V) the off current is higher than expected and around 1mA. Workaround: Unplug the battery when drones are not used and don’t just use the power off button.
- When the Bolt 1.1 is running on one cell battery (3-4.2V) don’t connect USB at the same time as the battery as the USB is leaking current and charging the battery unintentionally which potentially damages the battery after a while. Workaround: Don’t have the battery and the USB connected at the same time.

In [this](https://www.bitcraze.io/2020/10/testing-crazyflie-bolt-and-1-cell-li-ion/) blog post we powered it with 1-cell Li-ion.
